modules/ztp-configuring-spoke-at-remote-site.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="configuring-edge-cluster-at-remote-location_{context}"]
+= Configuring the edge cluster at the remote location
+
+Perform the following steps at the remote location.
+
+.Prerequisites
+
+* Follow guidance in the supplied quick start guide
```

Comment:
We need to make sure that either the quick start guide or this documentation describes **setting laptop's IP address/network and DNS server on the `master-0` of the cluster**.
That DNS resolves our initial domain to actual IP. The initial domain is the same for all boxes set by factory.